CONTRIBUTING.MD

```diff
@@ -42,12 +42,19 @@ If you would like to contribute, please follow the following steps:
 1. Fork the project.
 2. Clone your fork: `git clone <your_username>/gval`
 3. Create a feature branch: `git checkout -b <your_feature>`
-4. Code Standards: Make sure unit tests pass `pytest` and there is no significant reduction
-in code coverage `pytest --cov=gval ./tests`.
-5. Commit your changes: `git commit -m 'feature message'`  This will invoke pre-commit hooks that will lint
+4. Install the code via `pip install .`
+5. Set up pre-commit hooks via `pre-commit install`
+6. Code Standards: Make sure unit tests pass `pytest` and there is no significant reduction
+in code coverage and increase in memory.  Also run local_benchmark_test to make sure there is no significant
+cpu time performance loss.  To run everything (in root project directory):
+`pytest --memray --cov=gval --cov-report term-missing && python ./tests/local_benchmark.py`.
+7. If any changes were made to the documentation, make sure to run `python docs/compile_readme.py`.
```

Comment:
Is this for readme or documentation?

Reply:
It might be good to say that the readme is made up of other docs. These docs should be edited directly then the readme compiled?

Reply:
This is for both, I made changes to let the user know about the Markdown files.